pkg/scheduler/api/resource_info.go

```diff
@@ -234,57 +234,72 @@ func (r *Resource) Multi(ratio float64) *Resource {
 
 // Less checks whether a resource is less than other
 func (r *Resource) Less(rr *Resource) bool {
-	lessFunc := func(l, r float64) bool {
-		return l < r
+	cmpFunc := func(l, r float64) int {
+		lf := new(big.Rat).SetFloat64(l)
+		rf := new(big.Rat).SetFloat64(r)
+		return lf.Cmp(rf)
 	}
 
-	if !lessFunc(r.MilliCPU, rr.MilliCPU) {
+	isEqual := true
+	cpuCmp := cmpFunc(r.MilliCPU, rr.MilliCPU)
```

Comment:
I think we can update MilliCPU, memory to big.Rat

Reply:
ok done, PTAL )